site/gatsby-site/playwright/e2e-full/apps/checklistIndex.spec.ts

```diff
@@ -78,174 +72,143 @@ test.describe('Checklists App Index', () => {
+    test('Should show delete buttons only for owned checklists', async ({ page, login }) => {
 
         const [userId] = await login(config.E2E_ADMIN_USERNAME, config.E2E_ADMIN_PASSWORD, { customData: { first_name: 'Test', last_name: 'User', roles: ['admin'] } });
 
-        await conditionalIntercept(
-            page,
-            '**/graphql',
-            (req: any) => req.body.operationName == 'findChecklists',
-            {
-                data: {
-                    checklists: [
-                        {
-                            about: '',
-                            id: 'fakeChecklist1',
-                            name: 'My Checklist',
-                            owner_id: userId,
-                            risks: [],
-                            tags_goals: [],
-                            tags_methods: [],
-                            tags_other: [],
-                        },
-                        {
-                            about: '',
-                            id: 'fakeChecklist2',
-                            name: "Somebody Else's Checklist",
-                            owner_id: 'aFakeUserId',
-                            risks: [],
-                            tags_goals: [],
-                            tags_methods: [],
-                            tags_other: [],
-                        },
-                    ],
-                },
+        await init({
+            aiidprod: {
+                checklists: [
+                    {
+                        about: '',
+                        id: 'fakeChecklist1',
+                        name: 'My Checklist',
+                        owner_id: userId,
+                        risks: [],
+                        tags_goals: [],
+                        tags_methods: [],
+                        tags_other: [],
+                    },
+                    {
+                        about: '',
+                        id: 'fakeChecklist2',
+                        name: "Somebody Else's Checklist",
+                        owner_id: 'user1',
+                        risks: [],
+                        tags_goals: [],
+                        tags_methods: [],
+                        tags_other: [],
+                    },
+                ],
             },
-            'findChecklists',
-        );
+        }, { drop: true });
 
         await page.goto(url);
 
-        await waitForRequest('findChecklists');
+        await expect(page.locator('[data-cy="checklist-card"]:first-child [data-testid="delete-risk"]').first()).toContainText('Delete');
 
-        await expect(page.locator('[data-cy="checklist-card"]:first-child button')).toContainText('Delete');
-        await expect(page.locator('[data-cy="checklist-card"]:last-child button')).not.toContainText('Delete');
+        // TODO: looks like this page is filtering checklists owned by the current user, so this will never pass
```

Comment:
☝️

Reply:
Yeah, we used to have all checklist visible, and I think we discussed giving users the ability to make some of their checklists public. This should be re-enabled then if indeed we do that.